src/collective/cover/tests/test_upgrades.py

```diff
@@ -364,3 +364,39 @@ def test_remove_orphan_annotations(self):
         # simulate state on previous version
         self._do_upgrade_step(step)
         self.assertNotIn(foo, annotations)
+
+    def test_simplify_layout(self):
```

Comment:
test upgrade step to move column-site to it's parent and remove data attribute (with layout-type in it)